apps/authentication-portal/src/main/webapp/select_org.jsp

```diff
@@ -0,0 +1,158 @@
+<%--
+  ~ Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+  ~
+  ~ WSO2 LLC. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+ --%>
+
+<%@ page import="org.owasp.encoder.Encode" %>
+<%@ page import="org.wso2.carbon.identity.application.authentication.endpoint.util.Constants" %>
+<%@ page import="java.io.File" %>
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ include file="includes/localize.jsp" %>
+<%@ include file="includes/init-url.jsp" %>
+
+<%
+
+    String idp = request.getParameter("idp");
+    String authenticator = request.getParameter("authenticator");
+    String sessionDataKey = request.getParameter(Constants.SESSION_DATA_KEY);
+    int orgCount = Integer.parseInt(request.getParameter("orgCount"));
+
+    String errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, "error.retry");
+    String authenticationFailed = "false";
```

Comment:
Why are we using a string here instead of boolean?

Reply:
Yes, it should be boolean. Addressed by this PR https://github.com/wso2/identity-apps/pull/3523. 

In this PR I had a requirement to send list of organization ids, names and descriptions to the JSP file. What I have done is grouped individually encoded id,name,description by commas and concat these sets using `:` and send as query from to the jsp file. Is there any recommended way to achieve that type of requirement.